arcade/standard/blockglitch_4/map.xml

```diff
@@ -0,0 +1,607 @@
+<map proto="1.4.2" game="Progress">
+<!-- #region meta -->
+<name>BlockGlitch 4</name>
```

Comment:
Make sure you add the slug of the previous map's name so it'll be counted as the same map.
```suggestion
<name>BlockGlitch 4</name>
<slug>blockglitch_4_hell_edition</slug>
```